textproc/libxml2/Portfile

```diff
@@ -6,8 +6,8 @@ PortGroup           clang_dependency 1.0
 # Please keep the version of the libxml2 and py-libxml2 ports the same.
```

Comment:
as stated here, please make sure to update the `py-libxml2` to the same version. I notice that the`livecheck` for that port doesn't show the latest version, but it's there... So please add the update for the Python bindings to this PR as well and ideally fix the `livecheck` in that port.